modules/nvme/README.md

```diff
@@ -3,6 +3,9 @@ title: "NVMe devices monitoring with Netdata"
 custom_edit_url: https://github.com/netdata/go.d.plugin/edit/master/modules/nvme/README.md
 description: "Monitors NVMe devices health metrics using the nvme CLI tool."
 sidebar_label: "NVMe devices"
+learn_status: "Published"
+learn_topic_type: "References"
+learn_rel_path: "References/Collectors references/Devices"
```

Comment:
Devices?

Reply:
Yes the category/family of the collector

Reply:
Changing it to _Storage_

Reply:
Ok, I find it too generic.